### PR TITLE
Fix Prettier/ESLint for components

### PR DESCRIPTION
**Pull request by akshaynthakur**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

### :cloud: :cloud: Description :cloud: :cloud:
[//]: # "REQUIRED - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
- Changes in BackButton.tsx, CategoryCard.tsx, LabeledTextInput.tsx, ReactButton.tsx
- Updated function declarations
- Fixed imports (duplicates, unused imports)
- Specified types for props (eg. `TextStyle` instead of `any`)

### Screenshots :calling:
[//]: # "REQUIRED for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! If you are making changes here, please CC: @leexesther at the bottom."

## :zap: How to review :zap:
[//]: # 'REQUIRED - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'
- Wasn't able to run the app to make sure the build still works (outstanding Android Studio issues), so please do this when reviewing!

## :seedling: Next steps :seedling:
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### Online sources :information_source:
[//]: # 'Optional - copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs :raised_hands:
[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

[//]: # 'This tags the project leader as a default. Feel free to change, or add on anyone who you should be in on the conversation.'
CC: @davidqing6432

### CODE REVIEW

File: src/components/BackButton.tsx [hunk 1]
```diff
@@ -1,15 +1,8 @@
 import { Pressable, StyleSheet, Text } from "react-native";
+import React from 'react';
```
Comment (david-qing) [inline src/components/BackButton.tsx]: to be able to remove this import statement, can i have you rebase to main? this can be done using `git rebase main`

**Review by david-qing — APPROVED** [review]
* made prettier changes on some files
* changed names in Category Card to make more clear
* fixed up props that did not need to be destructured from objects
